numba/cuda/cudadrv/devicearray.py

```diff
@@ -85,14 +85,15 @@ def __init__(self, shape, strides, dtype, stream=0, writeback=None,
             shape = (shape,)
         if isinstance(strides, int):
             strides = (strides,)
+        dtype = np.dtype(dtype)
```

Comment:
https://github.com/numba/numba/pull/5253/files#diff-42092935baa0ef2d821a2466760dab18R76 Needs an update? And it'd be good to get in a couple of tests, thanks.

Reply:
Hmm...the link doesn't seem to be working for me. What are you looking for?

Reply:
Also where should these tests go?

Reply:
Docstring for this function has:
```
        dtype
            data type as np.dtype.
```
which I think needs changing.

Reply:
> Also where should these tests go?

How about:
```
numba.cuda.tests.cudadrv.test_cuda_ndarray.TestCudaNDArray.test_devicearray
```

Reply:
Ok those make sense. Thanks Stuart! Will update accordingly 😄

Reply:
great, thanks

Reply:
Have updated these. Thanks! 😄